app/components/super-features/super-features-relations/super-features-relations.html

```diff
@@ -1,7 +1,7 @@
 <div class="super-features-relations-list-header">
   <h4>Child Pages</h4>
 
-  <div ng-if="relations.length > 0">
+  <div ng-if="article.child_types.length > 0">
```

Comment:
It is the other way around, the check is too high as it exists. No child_types == no relations necessarily.